faster_coco_eval/core/cocoeval.py

```diff
@@ -128,7 +128,7 @@ def _toMask(anns, coco):
             # modify ann['segmentation'] by reference
             for ann in anns:
                 rle = coco.annToRLE(ann)
-                ann["rle"] = rle
+                ann["segmentation"] = rle
```

Comment:
Can we separate rle and segmentation in data? I did this on purpose because I wanted to leave both options

Reply:
I see, for boundary iou that works as well. I updated the "segmentation" keys back to "rle". 

Note: Since the "boundary" entries produced by boundary_utils will always be rles, there is currently no need to check and convert them in _toMask. Should be no problem to add this but I don't see a need for the moment and prefer to keep things simple.